crates/binstalk/src/bins.rs

```diff
@@ -47,7 +48,7 @@ pub fn infer_bin_dir_template(data: &Data) -> Cow<'static, str> {
         name.to_string(),
     ];
 
-    let default_bin_dir_template = Cow::Borrowed("{ bin }{ binary_ext }");
+    let default_bin_dir_template = Cow::Borrowed("{ bin }{ binary-ext }");
```

Comment:
did that use to work with the `_` or was that a bug we never caught?

Reply:
That's a bug introduced #417